website/docs/index.html.markdown

```diff
@@ -373,6 +373,102 @@ The following arguments are used to configure the VMware Cloud Director Provider
 * `import_separator` - (Optional; *v2.5+*) The string to be used as separator with `terraform import`. By default
   it is a dot (`.`).
 
+* `ignore_metadata_changes` - (Optional; *v3.10+*) Use one or more of these blocks to ignore specific metadata entries from being changed by this Terraform provider
+  after creation or when they were created outside Terraform.
+  See ["Ignore Metadata Changes"](#ignore-metadata-changes) for more details.
+
+## Ignore metadata changes
+
+One or more `ignore_metadata_changes` blocks can be optionally set in the provider configuration, which will allow to ignore specific `metadata_entry`
+items during `plan`, `updates-in-place` and `destroy` (**not** `create`). This is useful, for example, to avoid removing metadata entries that were created
+by an external actor or after they were created by Terraform.
+
+~> Note that this feature is only considered when using the `metadata_entry` argument in the resources and data sources that support
+it. In other words, to ignore metadata when you are using the deprecated `metadata` argument, please use the native Terraform `lifecycle` block.
+
+~> Be aware that setting a `metadata_entry` in your Terraform configuration that matches any `ignore_metadata_changes` can produce inconsistent
+results, as the metadata will be stored in state but nothing will be done in VCD. You can control what to do in this situation with
+`conflict_action`, which can be `error`, `warn` or `none`.
+
+The available sub-attributes for `ignore_metadata_changes` are:
+
+* `resource_type` - (Optional) Specifies the resource type which metadata needs to be ignored. If set, the resource type must be one of:
+  *"vcd_catalog"*, *"vcd_catalog_item"*, *"vcd_catalog_media"*, *"vcd_catalog_vapp_template"*, *"vcd_independent_disk"*, *"vcd_network_direct"*,
+  *"vcd_network_isolated"*, *"vcd_network_isolated_v2"*, *"vcd_network_routed"*, *"vcd_network_routed_v2"*, *"vcd_org"*, *"vcd_org_vdc"*, *"vcd_provider_vdc"*,
+  *"vcd_storage_profile"*, *"vcd_vapp"*, *"vcd_vapp_vm"* or *"vcd_vm"*, which are the resources compatible with `metadata_entry`.
+* `object_name`- (Optional) Specifies the name of the object which metadata needs to be ignored. All object types are supported, except for
```

Comment:
As mentioned above, this should be `resource_name`

Reply:
Changed